converter/converter.py

```diff
@@ -16,12 +16,21 @@ def trigger_conversion(self, kwargs):
         return self.output
 
     def _check_for_required(self, kwargs):
-        # TODO flatten kwargs and check all keys // BUT WE DONT NEED TO SUPPORT THIS
-        missing_kwargs = [kwarg for kwarg in self.required_keywords if kwarg not in kwargs.keys()]
+        '''
+        This function checks two things:
+        (1) the kwargs contain all required fields
+        (2) all required fields have a value – Google pathways does not consider an empty string to be valid input.
+        '''
+        missing_kwargs = []
+        for kwarg in self.required_keywords:
+            if kwarg not in kwargs.keys():
+                missing_kwargs.append(kwarg)
+            elif kwargs[kwarg] == '' or kwargs[kwarg] == None:
```

Comment:
Do we have/did you add a unit test/regression test to cover this case when a kwarg is empty?

Reply:
Ah, I do not. Just empty strings. Will add!